lib/output.js

```diff
@@ -1185,6 +1200,21 @@ function OutputStream(options) {
         output.print(self.getValue());
     });
     DEFPRINT(AST_String, function(self, output){
+        var parent = output.parent(1);
```

Comment:
I know. So I repeat the question. How is `parent(1)` different from `parent(0)`?